_tools/fetch-team-data.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Script to fetch team data from GitHub. Call using:
+ *     node _tools/fetch-team-data.js
+ * @author Nicholas C. Zakas
+ */
+
+/* eslint camelcase: [error, { properties: never }] */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const fs = require("fs");
+const octokit = require("@octokit/rest")();
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+// filename to output to
+const filename = "./_data/team.json";
+
+// retrieve token from environment
+const { ESLINT_GITHUB_TOKEN } = process.env;
+
+// exit early if no token is available
+if (!ESLINT_GITHUB_TOKEN) {
+    throw new Error("Missing ESLINT_GITHUB_TOKEN.");
+}
+
+// Github IDs for teams
+const ALUMNI_TEAM_ID = "3005888";
+const TSC_TEAM_ID = "2060414";
+const COMMITTERS_TEAM_ID = "1054435";
+
+// lookup table mapping Github team IDs to JSON keys
+const teamIds = {
+    [TSC_TEAM_ID]: "tsc",
+    [COMMITTERS_TEAM_ID]: "committers",
+    [ALUMNI_TEAM_ID]: "alumni"
+};
+
+// final data structure
+const team = {
+    tsc: [],
+    alumni: [],
+    committers: []
+};
+
+//-----------------------------------------------------------------------------
+// Requirements
```

Comment:
Why does this say “Requirements” again?

Reply:
Oops, just a copy paste error.

Reply:
Oops, just a copy-paste error.

```suggestion
// Main
```